cibyl/sources/elasticsearch/api.py

```diff
@@ -203,10 +203,18 @@ def get_deployment(self, **kwargs):
                 ip_version = matches.group(1)
 
             # Check if necessary filter by IP version:
-            ip_version_argument = kwargs.get('ip_version').value
-            if ip_version_argument and \
-                    ip_version not in ip_version_argument:
-                continue
+            if 'ip_version' in kwargs:
```

Comment:
I would suggest checking the kwargs outside of the loop and then here simply having the second if

Reply:
Done. At this way we can avoid one check in every loop. Thx.